app/frontend/src/components/forms/SubmissionsTable.vue

```diff
@@ -113,7 +137,19 @@ export default {
     },
   },
   methods: {
-    ...mapActions('form', ['fetchForm', 'fetchSubmissions']),
+    ...mapActions('form', ['fetchForm', 'fetchSubmissions', 'getFormPermissionsForUser']),
+
+    checkFormManage() {
```

Comment:
Lets use the checkFormManage in permissionUtils so this list of permissions isn't duplicated accross the code (1 place to change it)

Reply:
checkManageForm() takes a simplified form object:
```
currentVersionId: ​​
description: ​​
id: ​​
idps: ​​
name: ​​
permissions: 
```
so I can build that object, but the problem I'm having is that this.permissions arent committed and available in the store until getFormPermissionsForUser(this.formId) has completed.
I'll have to take another look

Reply:
Ok i would just move on and just make that array a constant so it's not duplicated here and permissionUtils

Reply:
ok. i tried doing something else but have gone with this solution